src/parser/priest/shadow/modules/talents/AuspiciousSpirits.tsx

```diff
@@ -36,13 +36,13 @@ class AuspiciousSpirits extends Analyzer {
           <>
             {formatNumber(this.insanity)} Insanity generated.<br /><br />
 
-            The damage displayed is the additional damage you gained from taking this talent. The Spirits are doing roughly twice as much overall damage.
+            The damage displayed is the additional damage you gained from taking this talent.
           </>
         )}
       >
         <BoringSpellValueText spell={SPELLS.AUSPICIOUS_SPIRITS_TALENT}>
           <>
-            <ItemDamageDone amount={this.damage / SPIRIT_DAMAGE_MULTIPLIER} />
+            <ItemDamageDone amount={this.damage - this.damage / SPIRIT_DAMAGE_MULTIPLIER} />
```

Comment:
Wouldnt this always be 0? Or 0 Divided by a multiplier?

Since this.damage - this.damage is just 0

Reply:
order of operations ;) but yeah, parenthesis would be useful to clarify